src/mozilla_sec_eia/library/validation_helpers.py

```diff
@@ -1,9 +1,14 @@
 """Implement common utilities/functions for validating models."""
 
+import json
+import re
 from importlib import resources
 
 import pandas as pd
 
+from mozilla_sec_eia.models.sec10k.ex_21.inference import clean_extracted_df
```

Comment:
As we move towards this becoming a `pudl_models` repo, I think it would be best to keep everything in the `library` package totally generic and avoid referencing any specific models. Keeping all the models isolated in their own packages will make the code structure cleaner as the repo grows, and eventually allow us to run models in their own isolated environments so we don't have to deal with dependency conflicts between models.

I think `strip_down_company_names` and `fill_nulls_for_comparison` are pretty generic and can be left here, but `clean_ex21_validation_set` would probably be better moved into `src/mozilla_sec_eia/models/sec10k/ex_21` somewhere.